plutus-pab/src/Plutus/PAB/Db/Beam/ContractDefinitionStore.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE GADTs             #-}
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes        #-}
+{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE TypeApplications  #-}
+{-# LANGUAGE TypeFamilies      #-}
+{-# LANGUAGE TypeOperators     #-}
+
+{-
+
+'beam' handler for the 'ContractDefinitionStore' effect
+
+-}
+module Plutus.PAB.Db.Beam.ContractDefinitionStore
+  (handleContractDefinitionStore)
+  where
+
+import           Control.Monad.Freer                     (Eff, Member, type (~>))
+import qualified Data.Text                               as Text
+import           Database.Beam                           (all_, select)
+import           Plutus.PAB.Effects.Contract             (ContractDefinitionStore (..))
+import           Plutus.PAB.Effects.Contract.ContractExe (ContractExe (..))
+import           Plutus.PAB.Effects.DbStore              hiding (contractPath)
+
+mkRow :: ContractExe -> Contract
+mkRow (ContractExe {..}) = Contract (Text.pack contractPath)
+
+fromRow :: Contract -> ContractExe
+fromRow (Contract {..})  = ContractExe (Text.unpack _contractPath)
+
+handleContractDefinitionStore ::
+  forall effs.
+  ( Member DbStoreEffect effs
+  )
+  => ContractDefinitionStore ContractExe
```

Comment:
Ideally we'd have a type parameter instead of `ContractExe` (with some constraints to make the serialisation work), but this probably requires some changes in other places, so maybe it's better to leave it for another PR.